web/modules/custom/collection/config/install/core.entity_view_display.rdf_entity.collection.default.yml

```diff
@@ -141,13 +152,16 @@ content:
   label:
     label: hidden
     type: string
-    weight: 1
+    weight: 0
     settings:
       link_to_entity: false
     third_party_settings: {  }
+  rdf_entity_field_collection_content_top:
+    weight: 1
+    settings: {  }
+    third_party_settings: {  }
```

Comment:
What is this? Is it maybe a field group or something? Is this intentional to make this appear here?